### ISSUE TITLE
Layered Neural Rendering for Retiming People in Video

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#118]
# TL;DR
This model allows you to change the timing (start timing and speed of movement) of the people in the video. First, the each person, including occlusion, is separated from the background. Then the background and each person's information are combined into a feature, and combined together to generate. They are able to change the timing of the water splashes and other features that accompany the movements of the people.

 SIGGRAPH Asia 2020
![Pasted Graphic](https://user-images.githubusercontent.com/24524018/93548678-b1217700-f9a2-11ea-98eb-85b1dc9867c3.gif)

![image](https://user-images.githubusercontent.com/24524018/93548721-c8606480-f9a2-11ea-97f8-d39121e42380.png)
![image](https://user-images.githubusercontent.com/24524018/93548726-cb5b5500-f9a2-11ea-8532-184de5c7cd63.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2009.07833

# Submission Dates(yyyy/mm/dd)
2020/09/16

# Authors and institutions
Erika Lu, Forrester Cole, Tali Dekel, Weidi Xie, Andrew Zisserman, David Salesin, William T. Freeman, Michael Rubinstein
- Google Research
- University of Oxford

# Methods

# Results

# Comments
